tests/unit/testHarnessTest.js

```diff
@@ -0,0 +1,50 @@
+define([
+    '$',
+    'text!fixtures/pinny.html',
+    'iframeTestHarness'
+], function(_$, fixture, iframeTestHarness) {
+    var Pinny;
+    var modalCenter;
+    var $element;
+    var $;
```

Comment:
We probably should be assigning it in the `beforeEach` below. It's supposed to be $ from the iframe.